.github/workflows/main.yml

```diff
@@ -47,6 +47,13 @@ jobs:
       env:
         PYTHON_VERSION: ${{ matrix.python-version }}
 
+    - name: Install dev/master versions
+      run: |
+       source "$CONDA/etc/profile.d/conda.sh"
+       conda activate voila-tests
+       pip uninstall jupyter_server -y
+       pip install git+https://github.com/jupyter/jupyter_server@master
```

Comment:
We should be able to install the released version now?

Reply:
Good idea, I'll do another pass of this PR.